pyrobosim/pyrobosim/core/world.py

```diff
@@ -683,13 +687,18 @@ def check_occupancy(self, pose):
         :return: True if the pose is occupied, else False if free.
         :rtype: bool
         """
+        if self.occupancy_grid:
+            x, y = (pose.x, pose.y) if isinstance(pose, Pose) else (pose[0], pose[1])
+            x, y = self.occupancy_grid.xy_from_world(x, y)
+            return self.occupancy_grid.data[x, y] == 1
```

Comment:
Rather than this, it might be cleaner if we have a `self.occupancy_grid.check_occupancy(x, y)` method implemented?

Reply:
This sounds good, 
as for the naming something like  `is_occupied(x, y)`  would be better ?

Reply:
That name works for me as well!